src/debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -305,6 +305,28 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
             return debugConfiguration.configureAspNetCoreSsl;
         }
 
+        const launchSettingsPath = path.join(path.dirname(projectFile), 'Properties', 'launchSettings.json');
+
+        if (await this.fsProvider.fileExists(launchSettingsPath)) {
+            let launchSettingsString = await this.fsProvider.readFile(launchSettingsPath);
+            if (launchSettingsString.charCodeAt(0) === 0xFEFF) {
+                // Remove byte order mark if needed
+                launchSettingsString = launchSettingsString.slice(1);
+            }
+            const launchSettings = JSON.parse(launchSettingsString);
```

Comment:
If this fails to parse, I feel like we should just return `false` (aka assume it's not ssl) rather than display an error to the user. I assume they would get a better error down the line from some other source

This also applies if it succeeds to parse, but the object doesn't match the structure you expect below